### ISSUE TITLE
question: publish SauceLab result

### ISSUE THREAD

**Issue by douglasduteil** [ModuleLoader/es6-module-loader#426]
Hi @guybedford 

Are you still running tests on SauceLab ?
If yes, what do you think of publishing the result of them in the README with a badge like : 

[![Sauce Test Status][saucelabs-image]][saucelabs-url]

or 

[![Sauce Test Status][saucelabs-browser-matrix-image]][saucelabs-url]

[saucelabs-url]: https://saucelabs.com/u/dd-es6-module-loader
[saucelabs-image]: https://saucelabs.com/buildstatus/dd-es6-module-loader
[saucelabs-browser-matrix-image]: https://saucelabs.com/browser-matrix/dd-es6-module-loader.svg

**Comment by guybedford** [comment]
Thanks @douglasduteil for the suggestion, yes this would be great. I did previously disable this, but forgot to re-enable it. I just tried getting this going again but am hitting not authorized issues in https://travis-ci.org/ModuleLoader/es6-module-loader/jobs/76861005#L1360.

Any idea what the fix would be? I've verified that the `SAUCE_USERNAME` and `SAUCE_ACCESS_TOKEN` environment variables are correctly configured in Travis.
